modules/doc/content/getting_started/installation/build_petsc_and_libmesh.md.template

```diff
@@ -1,16 +1,22 @@
 ```bash
 cd {{PATH}}/moose/scripts
 export MOOSE_JOBS=6 METHODS=opt
-./update_and_rebuild_petsc.sh {{PETSC_ARGS1}} {{PETSC_ARGS2}}
-./update_and_rebuild_libmesh.sh {{LIBMESH_ARGS1}}
-./update_and_rebuild_wasp.sh {{WASP_ARGS1}}
+./update_and_rebuild_petsc.sh {{PETSC_ARGS1}} {{PETSC_ARGS2}} || return
+./update_and_rebuild_libmesh.sh {{LIBMESH_ARGS1}} || return
+./update_and_rebuild_wasp.sh {{WASP_ARGS1}} || return
```

Comment:
This should prevent folks who copy paste the whole code block instead of one line at time, when returning to the Discussion forum with errors beyond the first (petsc fails, yet they seem to be running update_and_rebuild_libmesh.sh)

Reply:
pretty cool

Reply:
ah! thanks for commenting. Reminds me to go and look for: I think I have ` || exit 1` elsewhere, and that is wrong... it should be ` || return` like this change.